doi-transformer/src/test/java/no/unit/nva/doi/transformer/CrossRefConverterTest.java

```diff
@@ -626,6 +686,19 @@ boolean hasRole(Contributor contributor) {
         return nonNull(contributor.getRole());
     }
 
+    private static Instant sampleCrossrefDateAsInstant() {
+        LocalDate date = LocalDate.of(EXPECTED_YEAR, EXPECTED_MONTH, EXPECTED_DAY);
+        return Instant.ofEpochSecond(date.toEpochDay() * 86400L);
```

Comment:
What is 86400L?

Reply:
86400 seconds in a day. Cool way to convert LocalDate to Instant. I have changed it to "date.atStartOfDay(ZoneId.systemDefault()).toInstant();"